auth-web/src/components/auth/common/PaymentMethods.vue

```diff
@@ -0,0 +1,173 @@
+                <div v-else
+                  v-html="payment.description">
+                </div>
+              </div>
+            </div>
+          </v-expand-transition>
+        </div>
+
+        <div class="ml-auto pl-8">
+        <v-btn
+          depressed
+          color="primary"
+          width="120"
+          class="font-weight-bold"
+          :outlined="!isPaymentSelected(payment)"
+          @click="selectPayment(payment)"
+          :aria-label="'Select' + ' ' + payment.title"
+        >
+          <span>{{(isPaymentSelected(payment)) ? 'SELECTED' : 'SELECT'}}</span>
+        </v-btn>
+        </div>
+      </div>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { Account, PaymentTypes } from '@/util/constants'
+import { Component, Emit, Mixins, Prop, Vue } from 'vue-property-decorator'
+import PADInfoForm from '@/components/auth/common/PADInfoForm.vue'
+
+const PAYMENT_METHODS = {
+  [PaymentTypes.CREDIT_CARD]: {
+    type: PaymentTypes.CREDIT_CARD,
+    icon: 'mdi-credit-card-outline',
+    title: 'Credit Card',
+    subtitle: 'Pay for transactions individually with your credit card.',
+    description: `You don't need to provide any credit card information with your account. Credit card information will be requested when you are ready to complete a transaction.`,
+    isSelected: false
+  },
+  [PaymentTypes.PAD]: {
+    type: PaymentTypes.PAD,
+    icon: 'mdi-credit-card-outline',
+    title: 'Pre-authorized Debit',
+    subtitle: 'Automatically debit a bank account when payments are due.',
+    description: '',
+    isSelected: false
+  },
+  [PaymentTypes.BCOL]: {
+    type: PaymentTypes.BCOL,
+    icon: 'mdi-credit-card-outline',
+    title: 'BC Online',
+    subtitle: 'Use your linked BC Online account for payment.',
+    description: '',
+    isSelected: false
+  },
+  [PaymentTypes.ONLINE_BANKING]: {
+    type: PaymentTypes.ONLINE_BANKING,
+    icon: 'mdi-bank-outline',
+    title: 'Online Banking',
+    subtitle: 'Pay for products and services through your financial institutions website.',
+    description: `
+        <p>
+          Instructions to set up your online banking payment solution will be available in the <strong>Payment Methods</strong> section of your account settings once your account has been created.
+        </p>
+        <p class="mb-0">
+          BC Registries and Online Services <strong>must receive payment in full</strong> from your financial institution prior to the release of items purchased through this service. Receipt of an online banking payment generally takes 3-4 days from when you make the payment with your financial institution.
+        </p>`,
+    isSelected: false
+  }
+}
+
+@Component({
+  components: {
+    PADInfoForm
+  }
+})
+export default class PaymentMethodSelector extends Vue {
+  @Prop({ default: '' }) currentOrgType: string
+  private selectedPaymentMethod: string = ''
+  private paymentTypes = PaymentTypes
+  private paymentsPerAccountType = {
```

Comment:
can it be in a more common place?if it makes sense

Reply:
Yes, this I think we can move to either constants or config-helper file.